api/src/routing.js

```diff
@@ -111,6 +112,16 @@ app.postJsonOrCsv = (path, callback) => handleJsonOrCsvRequest('post', path, cal
 app.postJson = (path, callback) => handleJsonRequest('post', path, callback);
 app.putJson = (path, callback) => handleJsonRequest('put', path, callback);
 
+app.use(prometheusMiddleware({
+  metricsPath: '/api_prometheus_metrics',
```

Comment:
not sure where to put this. somewhere under `/api/`?

Reply:
I have moved to `/api/vi/prometheus_metrics`.

Can you elaborate on the risk of exposing PHI? I want to make sure I'm clear about your concerns.

Reply:
Cool, you're correct, there no immediate concern for PHI being exposed through this API at this moment. However, we should still be vigilant whenever we update it.